.pytool/CISettings.py

```diff
@@ -149,6 +169,18 @@ def GetActiveScopes(self):
                     scopes += ("gcc_arm_linux",)
                 if "RISCV64" in self.ActualArchitectures:
                     scopes += ("gcc_riscv64_unknown",)
+
+            try:
+                scopes += codeql_helpers.get_scopes(self.codeql)
```

Comment:
Should the adding of the scopes be inside of the if self.codeql check?

Reply:
That's handled in [`get_scopes()`](https://github.com/microsoft/mu_basecore/blob/release/202208/.pytool/Plugin/CodeQL/integration/stuart_codeql.py#L44-#L51).